libs/domains/services/feature/src/lib/service-links-popover/service-links-popover.tsx

```diff
@@ -48,8 +49,10 @@ export function ServiceLinksPopover({
   }
 
   return (
-    <Popover.Root>
-      <Popover.Trigger className={filteredLinks.length === 0 ? 'hidden' : ''}>{children}</Popover.Trigger>
+    <Popover.Root modal={true}>
+      <Tooltip content="Links">
```

Comment:
is it weird to have a tooltip `links` in hover when it's already written `links` on the btn?

Reply:
Fixed 
https://github.com/Qovery/console/pull/1132/commits/9985e03a0138fb0a09c38377004de6acd8906971